x-pack/plugins/infra/public/components/asset_details/tabs/processes/processes.tsx

```diff
@@ -119,23 +120,19 @@ export const Processes = () => {
               </EuiTitle>
             </EuiFlexItem>
             <EuiFlexItem grow={false}>
-              <EuiIconTip
+              <Popover
```

Comment:
I wonder if it makes sense to put the whole popover into the dedicated component instead of just its content. it would declutter a bit the parent component, and strictly speaking, there is no need to have the content separately as we don't use it anywhere else. wdyt?

Reply:
Yes in this case it makes sense to keep the `Popover` inside the other component as the content is not used anywhere else 👍  Changed